torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -745,6 +745,19 @@ RegisterOperators reg(
            handler(ss.str());
          },
          aliasAnalysisSpecialCase()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA("prim::Concat(...) -> Tensor"),
```

Comment:
So, the convention here is that it's (Tensors..., dim) right ?

Reply:
Yes. Added a comment above.